packages/connect-react/src/components/append/AppendAfterErrorScreen.tsx

```diff
@@ -0,0 +1,101 @@
+import log from 'loglevel';
+import React, { useCallback, useState } from 'react';
+
+import useAppendProcess from '../../hooks/useAppendProcess';
+import useShared from '../../hooks/useShared';
+import { AppendScreenType } from '../../types/screenTypes';
+import Checkbox from '../shared/Checkbox';
+import { PasskeyIssueIcon } from '../shared/icons/PasskeyIssueIcon';
+import { LinkButton } from '../shared/LinkButton';
+import { Notification } from '../shared/Notification';
+import { PrimaryButton } from '../shared/PrimaryButton';
+
+const AppendAfterErrorScreen = ({ attestationOptions }: { attestationOptions: string }) => {
+  const { config, navigateToScreen } = useAppendProcess();
+  const [error, setError] = useState<string | undefined>(undefined);
+  const [loading, setLoading] = useState(false);
+  const [dontShowAgain, setDontShowAgain] = useState(false);
+  const { getConnectService } = useShared();
+
+  const handleSubmit = useCallback(async () => {
+    if (loading) {
+      return;
+    }
+
+    setLoading(true);
+    setError(undefined);
+
+    const res = await getConnectService().completeAppend(attestationOptions);
+    if (res.err) {
+      log.error('error:', res.val);
+      setLoading(false);
+      setError('Passkey operation was cancelled or timed out.');
+
+      return;
+    }
+
+    if (dontShowAgain) {
+      const createEventRes = await getConnectService().recordEventUserAppendAfterLoginErrorBlacklisted();
+
+      if (createEventRes?.err) {
+        log.error('error:', createEventRes.val);
+      }
+    }
+
+    setLoading(false);
+    navigateToScreen(AppendScreenType.Success);
+  }, [getConnectService, config, navigateToScreen, loading]);
+
+  const handleSkip = async () => {
+    if (dontShowAgain) {
+      const createEventRes = await getConnectService().recordEventUserAppendAfterLoginErrorBlacklisted();
+
+      if (createEventRes?.err) {
+        log.error('error:', createEventRes.val);
+      }
+    }
+
+    config.onSkip();
+  };
+
+  const toggleDontShowAgain = () => setDontShowAgain(prev => !prev);
+
+  return (
+    <div className='cb-append-after-error-container'>
+      <div className='cb-h1'>Issues using passkeys?</div>
```

Comment:
Usually, we use cb-h2 for the first headline => I think we should use h2 here as well (also for the other screen)